src/Engine/Audio/SteamAudioInterface.hpp

```diff
@@ -1,130 +1,128 @@
 #pragma once
 
-#include <phonon.h>
-#include <vector>
 #include <cstdint>
-#include "SteamAudio.hpp"
+#include <vector>
+#include <phonon.h>
 #include "../linking.hpp"
+#include "SteamAudio.hpp"
 
 /// Interface to set up and send data to/from Steam Audio from the rest of the engine.
 class SteamAudioInterface {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') class 'SteamAudioInterface' does not have a copy constructor which is recommended since the class contains a pointer to allocated memory. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AnoCopyConstructor)